CHANGES.md

```diff
@@ -1,3 +1,129 @@
+### v0.15.1
+
+-  **TreeController**
+
+   Updates to the new version.
+
+-  **Component**
+
+   Makes creating Component’s templateObjects lazy and on-demand.
+   Reduces memory consumption.
+
+-  **Core**
+
+   Fixes super infinite loop in observed getter/setters.
+
+-  **DocumentResources**
+
+   Fix a bug affecting Firefox that was allowing components to be entering the document before its CSS stylesheets were loaded.
```

Comment:
Fix a bug affecting Firefox that was allowing components to be entering the document before _their_ CSS stylesheets were loaded.